apache/rocketmq/v2/service.proto

```diff
@@ -80,6 +84,8 @@ message ReceiveMessageRequest {
   int32 batch_size = 4;
   // Required if client type is simple consumer.
   optional google.protobuf.Duration invisible_duration = 5;
+  // For message auto renew and clean
+  bool auto_renew = 6;
```

Comment:
What is the rationale for this change?

Reply:
The field is used to distinguish messages' renew and clean logic. And the server will know whether to manage messages by request other than looking up in `Settings`.

Reply:
If I am guessing correctly, you are assuming this field is a flag instructing proxies to extend the invisible duration of messages on behalf of clients. 

It would be kind of strange adding this flag to this RPC. Would it be better to add this flag to the receive-message-RPC?